rllib/core/models/catalog.py

```diff
@@ -313,11 +407,11 @@ def get_dist_cls_from_action_space(
         *,
         framework: Optional[str] = None,
         deterministic: Optional[bool] = False,
-    ) -> Mapping[str, Any]:
+    ) -> Distribution:
         """Returns a distribution class for the given action space.
 
         You can get the required input dimension for the distribution by calling
-        `action_dict_cls.required_model_output_shape(action_space, model_config_dict)`
+        `action_dict_cls.required_input_dim(action_space)`
```

Comment:
@kouroshHakha As you pointed out once, we don't have to have shapes here. Further more, the multi distributions would become quite complex because the whole "input_lens" logic around splitting inputs would not work anymore.